policyengine_us/parameters/gov/states/pa/tanf/income/earned_deduction/earned_income_disregard/wed_flat.yaml

```diff
@@ -0,0 +1,11 @@
+description: PA deducts this amount from each adult budget member’s earned income when the individual is eligible for the Earned Income Disregard (EID).
+values:
+  2023-01-01: 200
+
+metadata:
+  unit: currency-USD
+  period: month
```

Comment:
make yearly

Reply:
or was there a reason why we left it monthly?

Reply:
We keep it in monthly format because the policy handbook was organized in month period, and we set up all test cases in monthly income.